src/test/java/seedu/duke/command/ArchiveNoteCommandTest.java

```diff
@@ -0,0 +1,134 @@
+package seedu.duke.command;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.duke.data.notebook.Note;
+import seedu.duke.data.notebook.Notebook;
+import seedu.duke.data.notebook.NotebookStub;
+import seedu.duke.data.notebook.Tag;
+import seedu.duke.ui.FormatterStub;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ArchiveNoteCommandTest {
+    int maxRowLength = 100;
+
+    Notebook notebook;
+    ArrayList<String> content;
+
+    ArrayList<Tag> tag = new ArrayList<>();
+    ArrayList<Tag> tagSet = new ArrayList<>();
+    Tag tagImpt;
+    Tag tagCs2113;
+    Tag tagNus;
+
+    @BeforeEach
+    void setup() {
+        notebook = new Notebook();
+        content = new ArrayList<>();
+        tagImpt = new Tag("Impt", Tag.COLOR_RED_STRING);
+        tagCs2113 = new Tag("CEG", Tag.COLOR_YELLOW_STRING);
+        tagNus = new Tag("NUS", Tag.COLOR_BLUE_STRING);
+
+        content.add("default");
+        content.add("hi how are you");
+
+        tag.add(tagImpt);
+
+        tagSet.add(tagNus);
+        tagSet.add(tagCs2113);
+
+        Note testNote1 = new Note("Default", content, true, false, tag);
+        Note testNote2 = new Note("TestNote1", content, false, false);
+        Note testNote3 = new Note("TestNote2", content, false, false, tagSet);
+        Note testNote4 = new Note("Random Text", content, true, false, tagSet);
+
+        notebook.addNote(testNote1);
+        notebook.addNote(testNote2);
+        notebook.addNote(testNote3);
+        notebook.addNote(testNote4);
+    }
+
+    @Test
+    void execute_validIndex_returnsArchiveMessage() {
+        int index = 1;
+        String title = NotebookStub.getArchiveNoteTitle(index);
+
+        String expected = "=".repeat(maxRowLength)
+                + FormatterStub.encloseRow(title)
+                + "=".repeat(maxRowLength)
+                + System.lineSeparator();
```

Comment:
All expected string can use the Formatter functions, for this case encloseString(title). Since the purpose of the command test are to compare the result and not the formatting, it is ok to just use those functions.